ui/src/pages/Admin/AllStories/index.tsx

```diff
@@ -235,24 +235,25 @@ export const AllStories: React.FC = () => {
     undefined
   );
   const classes = useStyles();
-  const allStoriesLabel = `${"ALL STORIES"} ${"("} ${
+  const allStoriesLabel = `${"ALL STORIES"} ${"("}${
```

Comment:
why not just have it like this: ``All STORIES (${state.origTableData.length})`` so much cleaner